src/main/resources/db/migration/v0.38.0.30__agm_agm_association_loads_2.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX agmagmassociation_agmassocobject_in ON public.agmagmassociation USING btree (agmAgmAssociationObject_id);
```

Comment:
This should be called _index like all the other indexes we have in the database.

Reply:
that index gets dropped in the FW file
v0.38.0.31__agm_agm_association_loads_3.sql